docs/source/guides/writer/chapters/writing.rst

```diff
@@ -60,7 +60,7 @@ Note that the test class provides you with a number of convenience attributes:
 
 To minimize the accidental clashes we define the public ones as properties
 so if you see something like ``AttributeError: can't set attribute`` double
-you are not overriding these.
+check you are not overriding these.
```

Comment:
Good catch.  I spotted that on line 59 there's a missing `:mod:` tag, and it should be:

```* And many more (see :mod:`avocado.core.test.Test`)```

Reply:
great catch I will add that.